content/0-setup/_index.md

```diff
@@ -0,0 +1,96 @@
++++
+title = "Setup"
+weight = 1
+chapter = true
++++
+
+# Setup
+
+{{%attachments style="grey" /%}}
```

Comment:
this line causes an error in my hugo build please remove 
```Error: Error building site: "intro-unix-shell/content/0-setup/_index.md:9:1": failed to extract shortcode: template for shortcode "attachments" not found```

Reply:
hm that's odd, the attachments shortcode is part of the learn theme so it shouldn't cause an error. I'll look into this

Reply:
im probably doing something dumb then. Ill let you know if i fix it

Reply:
learn wasn't installed as it wasn't added as a submodule. My bad. I might want to fix this in the template though at some point